src/ExNavigationBar.js

```diff
@@ -394,6 +399,14 @@ const styles = StyleSheet.create({
   appbarTranslucent: {
     backgroundColor: 'rgba(255,255,255,0.7)',
   },
+  backgroundImage: {
+    top:0,
+    left:0,
+    right:0,
+    position: 'absolute',
+    resizeMode: 'cover', 
+    width: window.width,
```

Comment:
This should not be necessary. We should otherwise figure out a way to not use `Dimesnions` here.